src/api/app.js

```diff
@@ -345,6 +357,15 @@ module.exports.isNetStateDisconnected = (appHandle) => {
  * @param {SAFEAppHandle} appHandle
  *
  * @returns {Boolean}
+ *
+ * @example // Resets native object cache as well as front-end object handles except for SAFEApp
+ * window.safeApp.clearObjectCache(appHandle)
+ *   .then(() => {
+ *     console.log('All network objects dropped');
+ *     console.log(appHandle, ' still valid.);
+ *     window.safeApp.isRegistered(appHandle)
+ *     .then(console.log);
```

Comment:
Nice example, although this last line will log just a boolean, perhaps improve it to show a message that says something about that value.